x-pack/plugins/task_manager/README.md

```diff
@@ -162,6 +178,39 @@ The task runner's `run` method is expected to return a promise that resolves to
 
 Other return values will result in a warning, but the system should continue to work.
 
+### Task retries when the Task Runner fails
+If a task runner throws an error, task manager will try to rerun the task shortly after (up to the task definition's `maxAttampts`).
+Normal tasks will wait a default amount of 5m before trying again and every subsequent attempt will add an additonal 5m cool off period to avoid a stampeding herd of failed tasks from storming Elasticsearch.
+
+Recurring tasks will also get retried, but instead of using the 5m interval for the retry, they will be retried on their nex tscheduled run.
```

Comment:
Just looking at the changes in this PR and noticed this typo: `their nex tscheduled run.` should be `their next scheduled run.`

Reply:
Thanks Devin :)